src/text/skeleton.scss

```diff
@@ -3,21 +3,26 @@
 @import '../style/index';
 
 @mixin iui-text-skeleton {
-  display: inline-block;
```

Comment:
hm...
Sure about this removal?
Thats the reason to that issue in my other comment about same widths.

Reply:
It's put back now.